src/components/Palette/Palette.scss

```diff
@@ -0,0 +1,35 @@
+@use '../variables';
+
+$block: '.#{variables.$ns}palette';
+
+#{$block} {
+    display: inline-flex;
+    flex-flow: column wrap;
+    gap: 8px;
+
+    &:focus {
+        border: none;
+        outline: none;
+    }
+
+    &__row {
+        display: inline-flex;
+        gap: 8px;
+    }
+
+    &_size_xs &__option {
+        font-size: 16px;
+    }
+    &_size_s &__option {
+        font-size: 18px;
+    }
+    &_size_m &__option {
+        font-size: 20px;
+    }
+    &_size_l &__option {
+        font-size: 24px;
+    }
+    &_size_xl &__option {
+        font-size: 28px;
```

Comment:
We need shift the sizes down, they're a bit bigger than they should be. You can check font sizes of Button

Reply:
Shifted the font sizes down according to the `Button`'s `--_--icon-size`: `12px`/`16px`/`16px`/`16px`/`20px`
The font sizes of `Button` are basically the same (except for the `xl`), so I didn't use them.